setup.py

```diff
@@ -16,7 +16,7 @@
     url='http://github.com/praekelt/django-recaptcha',
     packages=find_packages(),
     install_requires=[
-        'django>1.11,<3.0',
+        'django>1.11,<4.0',
```

Comment:
**Suggestion** We can probably remove the upper cap. The use of upper caps should be limited to cases where we know we are not compatible. If we set it too soon, then we make it hard/impossible to discover the compatibility issues or would require us to be very fast to test with every new version (which I doubt we are going to be after we finally for the project back on the road).